src/runtime_bignum_test.nr

```diff
@@ -123,40 +319,62 @@ fn test_add<let N: u32, Params>(bn: BigNumInstance<N, Params>) where Params: Big
 }
 
 fn test_div<let N: u32, Params>(bn: BigNumInstance<N, Params>) where Params: BigNumParamsTrait<N> {
-    let a: BigNum<N, Params> = bn.__derive_from_seed([1, 2, 3, 4]);
-    let b: BigNum<N, Params> = bn.__derive_from_seed([4, 5, 6, 7]);
+    let a: BigNum<N, Params> = unsafe {
+        bn.__derive_from_seed([1, 2, 3, 4])
+    };
+    let b: BigNum<N, Params> = unsafe {
+        bn.__derive_from_seed([4, 5, 6, 7])
+    };
 
     let c = bn.div(a, b);
     assert(bn.eq(bn.mul(b, c), a));
 }
 
 fn test_invmod<let N: u32, Params>(bn: BigNumInstance<N, Params>) where Params: BigNumParamsTrait<N> {
-    let u: BigNum<N, Params> = bn.__derive_from_seed([1, 2, 3, 4]);
+    let u: BigNum<N, Params> = unsafe {
+        bn.__derive_from_seed([1, 2, 3, 4])
+    };
     for _ in 0..1 {
-        let v = bn.__invmod(u);
-        let result = bn.__mul(u, v);
+        let v = unsafe {
+            bn.__invmod(u)
+        };
+        let result = unsafe {
+            bn.__mul(u, v)
+        };
         let expected: BigNum<N, Params> = BigNum::one();
         assert(result.limbs == expected.limbs);
     }
 }
 
 fn assert_is_not_equal<let N: u32, Params>(bn: BigNumInstance<N, Params>) where Params: BigNumParamsTrait<N> {
```

Comment:
Is this func used anywhere now that we have `assert_is_not_equal` in the macro test with 
```
#[make_test(quote{3},quote{BNParams})]
pub struct BNTests{}
```
Seems we can delete some of these functions that take in a `BigNumInstance` as inputs.